bizhawk-make-install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+BASEDIR=$(dirname $0)
+if [ "$(uname -s)" = "Linux" ]; then
+	cd "$BASEDIR/src/platform/bizhawk/linux"
+else
+	cd "$BASEDIR/src/platform/bizhawk/mingw"
+fi
+make clean
+make -j4
```

Comment:
Recommendation:
```suggestion
make -j${JOBS:-4}
```

This lets you override the job number from the environment if desired, but defaults to 4 if it's not present.

Reply:
Gentoo and Arch have established a precedent for something like `make ${MAKEOPTS:-j4}`